### DISCUSSION TITLE
Unable to send images: Code 8

### DISCUSSION THREAD

**Discussion by c64whiz** [FossifyOrg/General-Discussion#499] [General]
Hello, I'm not opening an issue 'cause I'm pretty confident this isn't a problem with Messages but with a service I removed/uninstalled via "Universal Android Debloater".  My problem is when I send a picture in Messages, it doesn't go through but returns the error: "Couldn't send message, error code: 8".

My question is what does the error code mean?  It may give me a lead as to what service needs to be restored/re-installed.

TIA.
![pic](https://github.com/user-attachments/assets/e9c41b13-57ef-488c-87ef-12ee4202b926)

**Comment by naveensingh** [comment]
There are similar issues related to MMS but error code 8 seems different as per the docs:

```kotlin
    /** Failed because the user has denied this app ever send premium short codes. */
    public static final int RESULT_ERROR_SHORT_CODE_NEVER_ALLOWED = 8;
```

I would suggest you to re-enable the disabled system apps and try again. If I remember correctly, UAD has an export option.
